che-theia-plugins.yaml

```diff
@@ -768,3 +768,11 @@ plugins:
     repository:
       url: 'https://github.com/BroadcomMFD/cobol-control-flow'
     extension: https://open-vsx.org/api/BroadcomMFD/ccf/0.4.0/file/BroadcomMFD.ccf-0.4.0.vsix
+  - repository:
+      url: 'https://github.com/randomchance/vscode-logstash-configuration-syntax'
+      revision: '0.0.4'
```

Comment:
I don't see 0.0.4 tag in https://github.com/randomchance/vscode-logstash-configuration-syntax. Please see what revision attribute should describe: https://github.com/eclipse-che/che-plugin-registry/blob/main/CONTRIBUTE.md

Reply:
You are right. I change revision by commit number